NOTICE

```diff
@@ -39,3 +39,15 @@ Copyright 2021 The Flink Remote Shuffle Project
 
 This product includes software developed at The Alibaba Group Holding Ltd
 (https://www.alibaba.com/).
+
+MyBatis
```

Comment:
only NOTICE-binary should be updated, because we don't copy the source code from MyBatis